Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM python:3.11-slim
+LABEL maintainer="sanyok.it@gmail.com"
+
+ENV PYTHONDONTWRITEBYTECODE=1
+
+WORKDIR /app
+
+COPY requirements.txt requirements.txt
+RUN pip install --no-cache-dir -r requirements.txt
+
+COPY app/ /app/
```

Comment:
The `COPY app/ /app/` command assumes that there is an `app` directory in the build context. Ensure that this directory exists and contains the necessary application files.